lib/bootloader_setup.pm

```diff
@@ -1001,6 +1002,11 @@ sub tianocore_select_bootloader {
     send_key 'ret';
 }
 
+sub tianocore_set_xga_resolution {
+    tianocore_enter_menu;
+    tianocore_ensure_xga_resolution if check_var('QEMUVGA', 'qxl');
```

Comment:
IMO this can be done unconditionally. Otherwise we could have a `generate_pflash` test which doesn't do anything.

Is there any `QEMUVGA` where this might make it worse?